ddtrace/internal/datadog/profiling/include/interface.hpp

```diff
@@ -0,0 +1,47 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0. This product includes software
+// developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present
+// Datadog, Inc.
+
+#pragma once
+
+#include "exporter.hpp"
+
+extern "C" {
```

Comment:
I think pyx has support for c++ ?

Reply:
Indeed, it was just annoying to refactor everything in this PR.  It's definitely my plan to fix this.